src/overlay/test/LoopbackPeer.h

```diff
@@ -35,6 +35,15 @@ class LoopbackPeer : public Peer
 
     bool mDamageCert{false};
     bool mDamageAuth{false};
+
+    // Test tx set flooding by suspending tx flooding.
+    // If we allow tx flooding, it is possible that
+    // nodes don't need to flood tx sets because
+    // nodes may have the exact same txns
+    // in their TransactionQueue.
+    // This is more likely to be true
+    // esp in test environments with not many real-world obstacles.
+    bool mSuspendTxFlooding{false};
```

Comment:
that's an interesting idea! I agree tx flooding can prevent us from flooding what we want.